skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/ChatScreen.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2024 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.tooling.aibot
+
+import com.slack.circuit.runtime.CircuitUiEvent
+import com.slack.circuit.runtime.CircuitUiState
+import com.slack.circuit.runtime.screen.Screen
+
+data object ChatScreen : Screen {
+  data class State(
+    val messages: List<Message>,
+    val currentInput: String,
+    val eventSink: (Event) -> Unit,
```

Comment:
nit: for testability, we usually default event sinks to `= {}` for ease of standing up new state objects

Reply:
I set it here: https://github.com/slackhq/slack-gradle-plugin/blob/f27ddc1af95d1c0b90e33622407588cd464def09/skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/ChatScreen.kt

with this commit: f27ddc1af95d1c0b90e33622407588cd464def09

I will make sure to separate my commits, sorry for the headache.